prance/util/resolver.py

```diff
@@ -111,6 +128,8 @@ def _dereferencing_iterator(self, base_url, partial, path, recursions):
       # Split the reference string into parsed URL and object path
       ref_url, obj_path = _url.split_url_reference(base_url, refstring)
 
+      translate = self.__resolve_method == TRANSLATE_EXTERNAL and self.parsed_url.path != ref_url.path
```

Comment:
The _translate_ variable is initialized here, but used only down there in the _yield_ condition. I know this is a remnant from the original approach, but now it is only confusing.